spec/features/create_dataset_spec.rb

```diff
@@ -178,7 +178,6 @@
 
         expect(page).to     have_content('Start Date')
         expect(page).to     have_content('End Date')
-        expect(page).to_not have_content('Year')
```

Comment:
Because they shouldn't have been there in the first place (Year should be present on all datafile forms (except daily and one-off). However the test still passed because the template had 'year' instead of 'Year'. When I corrected the capitalisation, that test started to fail, because of the test itself.